EventHorizon/Spell/Buffer.lua

```diff
@@ -91,6 +95,11 @@ end
 
 function Buffer:RemoveBuff(target)
 	self.buffs[target] = nil
+	-- manually stop indicators since self buffs can be right-clicked
+	local now = GetTime()
+	for _,indicator in pairs(self.indicators) do
```

Comment:
What is this segment of code actually doing? I don't see any conditionals that make the distinction between self-buff and everything else. This is just stopping all buff indicators for a particular buff for every instance of its application.